services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/create_unit/assignment_card.tsx

```diff
@@ -48,6 +52,7 @@ export default class AssignmentCard extends React.Component<AssignmentCardProps,
 
   render() {
     const { imgSrc, imgAlt, imgClassName, showNewTag, header, bodyArray, showRecommendedToStartTag, } = this.props
+    const isDisabled = DISABLED_DIAGNOSTICS.includes(header)
```

Comment:
I think that since we'll ultimately be replacing all the diagnostics shown on this page, and never showing the old diagnostics again once we do, that this could be a temporary constant that we get rid of when we update the IDs, but this approach works fine so no need to change anything unless you want to.